src/net.h

```diff
@@ -27,6 +27,7 @@
 #include "util.h" // FIXME: reduce scope
 
 #include <atomic>
+#include <boost/lexical_cast.hpp>
 #include <deque>
```

Comment:
why not also take it out of net.h as well, it's only used in one place on line 926, also could remove it from net.cpp where it isn't even used.